websites/R/Reddit/presence.ts

```diff
@@ -35,28 +36,56 @@ async function getStrings() {
 	);
 }
 
-const startTimestamp = Math.floor(Date.now() / 1000),
-	oldReddit = document.querySelector("._1tvdSTbdxaK-BnUbzUIqIY") === null;
+const presences: { [key in PresenceClients]?: Presence } = {
+		[PresenceClients.Reddit]: presence,
+	},
+	startTimestamp = Math.floor(Date.now() / 1000),
+	oldReddit = document.querySelector(".default-header") === null;
```

Comment:
Since this element appears on old reddit, the comparison is incorrect.
```suggestion
	oldReddit = document.querySelector(".default-header") !== null;
```
```suggestion
	oldReddit = !!document.querySelector(".default-header");
```